src/WSRequestHandler_SceneItems.cpp

```diff
@@ -421,12 +424,21 @@ RpcResponse WSRequestHandler::SetSceneItemRender(const RpcRequest& request) {
 		return request.failed("requested scene doesn't exist");
 	}
 
-	OBSSceneItemAutoRelease sceneItem =
-		Utils::GetSceneItemFromName(scene, itemName);
-	if (!sceneItem) {
-		return request.failed("specified scene item doesn't exist");
-	}
+	OBSSceneItemAutoRelease sceneItem;
 
+	if (strlen(itemName)) {
+		OBSDataItemAutoRelease itemField = obs_data_item_byname(request.parameters(), "source");
```

Comment:
What's the use for this variable? I don't see it used anywhere in the scope block.